pkg/minishift/openshift/openshift.go

```diff
@@ -70,12 +73,26 @@ func (t *OpenShiftPatchTarget) localConfigFilePath() (string, error) {
 }
 
 func RestartOpenShift(commander docker.DockerCommander) (bool, error) {
-	ok, err := commander.Restart(constants.OpenshiftContainerName)
+	var ok bool
```

Comment:
I would have good to have this form here
```
var (
    ok bool
    err error
)
```
More scalable but not blocking to merge.